frontend/src/components/studySamples/StudySamples.tsx

```diff
@@ -96,64 +95,81 @@ function StudySamples({ studyID, studySamples, refreshSamples }: StudySamplesPro
 			</div>
 			<Collapse bordered={true} onChange={handleExpand} activeKey={expandedPanelKeys}>
 				{renderedSteps.map((step) => {
-					const countString = `${step.completedCount} / ${step.sampleCount + step.completedCount}`
-					const countTitle = `${step.completedCount} of ${step.sampleCount + step.completedCount} samples are completed`
-
-					return (
-						<Collapse.Panel
-							key={step.stepID}
-							header={
-								<Space align="baseline">
-									<Text strong={true} style={{fontSize: 16}}>{step.stepOrder}</Text>
-									<Text>{step.stepName}</Text>
-								</Space>
-							}
-							showArrow={true}
-							extra={
-								<>
-									<Space>
-										<Title level={4} style={{ margin: '0' }} title={countTitle}>
-											{countString}
-										</Title>
-									</Space>
-								</>
-							}
-							style={{backgroundColor: 'white'}}
-						>
-							<SamplesTabs studyID={studyID} step={step} settings={uxSettings?.stepSettings[step.stepID]}/>
-						</Collapse.Panel>
-					)
+					// Call StepPanel as a function because the child of Collapse must be a CollapsePanel, not a StepPanel
+					return StepPanel({step, studyID, uxSettings:uxSettings?.stepSettings[step.stepID]})
 				})}
 			</Collapse>
 		</>
 	)
 }
 
-interface SampleTabContainerProps {
-	studyID: FMSId
+interface StepPanelProps {
 	step: StudySampleStep
-	settings?: StudyUXStepSettings
+	studyID: FMSId
+	uxSettings?: StudyUXStepSettings
 }
-function SamplesTabs({studyID, step, settings}: SampleTabContainerProps) {
+function StepPanel({step, studyID, uxSettings} : StepPanelProps) {
 	const dispatch = useAppDispatch()
 
+	const countString = `${step.completedCount} / ${step.sampleCount + step.completedCount}`
+	const countTitle = `${step.completedCount} of ${step.sampleCount + step.completedCount} samples are completed`
+	
+	const completedSamples = step.completed.filter(completed => completed.removedFromWorkflow === false)
+	const removedSamples = step.completed.filter(completed => completed.removedFromWorkflow === true)
+	const hasRemovedSamples = removedSamples.length > 0
+
+	const removedTitle = removedSamples.length === 1 ? `1 sample was removed from study` : `${removedSamples.length} samples were removed from study`
```

Comment:
I think we should specify in this message that the sample was removed after the step was completed. It is to make sure it is clear the sample is not missing from this step listing but will from further study steps.

Reply:
I added "at this step" to make it a bit more clear. We could also add an info message in the removed samples section to make it even more clear "These samples were removed from the study at this step and will not appear in any later steps."